package.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "metabase-nodejs-static-embedding-sample",
+    "homepage": "https://github.com/metabase/metabase-nodejs-static-embedding-sample",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "scripts": {
+      "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "author": "Metabase",
+    "license": "MIT",
+    "dependencies": {
+      "express": "^4.18.2",
+      "jsonwebtoken": "^9.0.2"
+    }
+  }
```

Comment:
No newlines anywhere. This could be fixed either in your editor, or in `.gitconfig` (https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings).

Reply:
I don't think so Alberto. The commit has been already made and pushed, so you'll have to save files with proper line endings and commit them again. Where these changes will originate from - gitconfig, your IDE or even manual editing - won't matter.